docs/usage_guide.rst

```diff
@@ -414,7 +418,20 @@ Additionally below the overview you get a table containing all of the taxable ev
    :align: center
 
 
-Finally you can get a nice CSV export by pressing the "Export CSV" button. This export is meant to be imported into google sheets. Press the button and then choose a directory to write the CSV files to. Once done you can import the CSV files into google sheets via its import menu.
+Finally you can get a CSV export by pressing the "Export CSV" button. This export is meant to be imported into Google Sheets. Press the button and then choose a directory to write the CSV files to. Once done you can import the CSV files into Google Sheets via its import menu. Following are definitions for the document's columns
+
+- ``type`` is a string describing the type of action a user engaged in.
+- ``paid_asset`` is a string identifying the asset an action was paid in, e.g. in "I bought 1 ETH for 100 EUR", `EUR` is the  `paid_asset`.
+- ``paid_in_asset`` is a float specifying the number of `paid_asset`s involved in an action.
+- ``taxable_amount``: is a float specifying the amount of `paid_asset` needed to be taken into consideration when generating a tax report.
+- ``received_asset`` is a string identifying the asset of an action's yield, e.g. in "I bought 1 ETH for 100 EUR", `ETH` is the `received_asset`.
+- ``received_in_asset`` is a float specifying the number of `received_asset`s involved in an action.
```

Comment:
Does it make sense to replace "number" with "amount" here? (...specifying the _amount_ of ...) Additionally, since you've done the work of providing a nice example I would suggest continuing with it through the documentation (where applicable)

e.g.

> `paid_in_asset`: is a float specifying the amount of `paid_asset` involved in an action/event, e.g. in "100" in the above example.

Reply:
> Does it make sense to replace "number" with "amount" here? (...specifying the amount of ...)

Replaced it.

> Additionally, since you've done the work of providing a nice example I would suggest continuing with it through the documentation (where applicable)

I've done it for a couple now. But I found some too complex to make an example... Hope that's OK.